substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -2325,20 +2344,20 @@ pub mod pallet {
 
 				if member_account == bonded_pool.roles.depositor {
 					Pallet::<T>::dissolve_pool(bonded_pool);
-					None
+					Weight::default()
```

Comment:
This is just a refactor and keeps the same weight logic as before. 

But good question. I am guessing we want to incentivise dissolving of pool so its free :D. It would remove items from storage so its a good thing for the chain.